client/src/i18n/en/stock.json

```diff
@@ -103,13 +103,13 @@
     "MONTHLY_CONSUM"  : "Average Monthly Consumption",
     "MONTHLY_CONSUMPTION" : {
       "ALGO_1"          : "Algorithm 1",
-      "ALGO_1_COMMENT"  : "The average monthly consumption is obtained by dividing the quantity consumed during the period defined by the number of days of stock for said period, and by multiplying the result obtained by 30.5",
+      "ALGO_1_COMMENT"  : "The average monthly consumption is obtained by dividing the quantity consumed during the period by the number of days having stock for the period, and by multiplying the result by 30.5.",
       "ALGO_2"          : "Algorithm 2",
-      "ALGO_2_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period defined by the number of days of consumption for said period, and by multiplying the result obtained by 30.5",
+      "ALGO_2_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period by the number of days of consumption for the period, and by multiplying the result by 30.5.",
       "ALGO_3"          : "Algorithm 3",
-      "ALGO_3_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period defined by the number of days in the period, and by multiplying the result obtained by 30.5",
+      "ALGO_3_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period by the number of days in the period, and by multiplying the result obtained by 30.5.",
       "ALGO_4"          : "Algorithm 4 (MSH)",
-      "ALGO_4_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period defined by the number of months minus the number of months divided by the total number of days of rupture"
+      "ALGO_4_COMMENT"  : "The average consumption is obtained by dividing the quantity consumed during the period by the difference of number of months in the period minus the total number of days of stock out in the period. The MSH algorithm is recommended by the Management Sciences for Health organization (https://www.msh.org)."
```

Comment:
I think we are missing "the" in "during the period by the difference of *the* number of months in the period".

Reply:
Fixed.